.ci/test.py

```diff
@@ -241,13 +241,14 @@ def run_one(p: Plugin, workflow: str) -> bool:
 
     logging.info(f"Virtualenv at {vpath}")
 
+    num_workers = 1 if p.name == "sauron" else 5
```

Comment:
@sip-21 didn't you fix the concurrency? We can get rid of this I think?

Reply:
If removing this breaks the tests, we should add the changes to this file back, but under the other commit (the one that adds the tests).

Reply:
Tried removing these lines from the code. Tests are all failing now.